packages/web/src/scss/components/Button/_Button.scss

```diff
@@ -50,6 +50,10 @@
     width: 100%;
 }
 
+.Button > svg {
+    pointer-events: none;
```

Comment:
Hmm 🤔 How about moving this to our foundation CSS?

```css
button > svg {
  pointer-events: none;
}
```

Reply:
I have no strong opinion on `>` here. But I'd expect the list of selectors to grow.